webapp/javascript/components/FlameGraph/FlameGraphComponent/Highlight.tsx

```diff
@@ -5,7 +5,7 @@ import styles from './Highlight.module.css';
 export interface HighlightProps {
   // probably the same as the bar height
   barHeight: number;
-
+  zoom: any;
```

Comment:
```suggestion
  zoom: Option<DeepReadonly<{ i: number; j: number }>>
```
Which is the same type as https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/components/FlameGraph/FlameGraphComponent/Flamegraph.ts#L41